recipes/asciigenome/meta.yaml

```diff
@@ -1,10 +1,10 @@
 package:
   name: asciigenome
-  version: '0.6.4'
+  version: '1.0.0'
 
 source:
-  fn: ASCIIGenome-0.6.4.zip
-  url: https://github.com/dariober/ASCIIGenome/releases/download/v0.1.0/ASCIIGenome-0.6.4.zip
+  fn: ASCIIGenome-1.0.0.zip
+  url: https://github.com/dariober/ASCIIGenome/releases/download/v1.0.0/ASCIIGenome-1.0.0.zip
```

Comment:
Can you please add an `md5:` checksum for this and use `ASCIIGenome-1.0.0.tar.gz`
Thanks!